boot/bootutil/src/ed25519_psa.c

```diff
@@ -30,7 +30,6 @@ static psa_key_id_t kmu_key_ids[3] =  {
     MAKE_PSA_KMU_KEY_ID(228),
     MAKE_PSA_KMU_KEY_ID(230)
 };
-#define KMU_KEY_COUNT (sizeof(kmu_key_ids)/sizeof(kmu_key_ids[0]))
```

Comment:
can we have assertion on limit CONFIG_BOOT_SIGNATURE_KMU_SLOTS value to KMU_KEY_COUNT as max?

Reply:
The limit is enforced by Kconfig

Reply:
Added a build assert